cookbook/python-node/main.py

```
@@ -1,4 +1,5 @@
 import asyncio
+from brainyflow import Memory # Import Memory
```

Ruff (0.8.2)

`brainyflow.Memory` imported but unused

Remove unused import: `brainyflow.Memory`

(F401)